save-backend/src/main/kotlin/com/saveourtool/save/backend/configs/WebConfig.kt

```diff
@@ -32,6 +33,7 @@ class WebConfig {
     fun jackson2ObjectMapperBuilderCustomizer() = Jackson2ObjectMapperBuilderCustomizer { jacksonObjectMapperBuilder: Jackson2ObjectMapperBuilder ->
         jacksonObjectMapperBuilder
             .mixIn(TestStatus::class.java, TestStatusMixin::class.java)
+            .featuresToDisable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
```

Comment:
why needed in Jackson?

Reply:
I passed creation time as LocalDateTime, but I changed to long. Will revert